Source.lua

```diff
@@ -528,7 +531,7 @@ local function luau_load(module, env, luau_settings)
 		luau_validatesettings(luau_settings)
 	end
 
-	if type(module) == "string" then
+	if type(module) == "string" or type(module) == "buffer" then
```

Comment:
Since a deserialized module must be a table, just checking if the module isn't a table would suffice.